lib/logic.rb

```diff
@@ -0,0 +1,73 @@
+class TicTacToe
+  attr_accessor :token, :player
+  attr_reader :board, :game, :player_char, :loc, :move, :winner
```

Comment:
- [ ] The student uses attr_readers/attr_writers. Only uses attr_accessors if they really need it and must be aware that by using it they allow writing that property from outside the class. That variable can be set to anything. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

Modify this to attr_reader and attr_writers unless you are sure that you need accessors please.